runner/src/storage/remote.ts

```diff
@@ -64,6 +64,12 @@ export interface RemoteStorageProviderSettings {
 }
 
 export interface RemoteStorageProviderOptions {
+  /**
+   * Unique identifier of the storage. Used as name of the `BroadcastChannel`
```

Comment:
could this be useful to send to common/memory as well to correlate things between the client and server?

Reply:
Server does not need to know about it, it's just for local updates.